src/utils/navigation.ts

```diff
@@ -154,5 +154,10 @@ export const NAVIGATION: Navigation = {
     currentPage: config.SIGN_OUT_PAGE,
     previousPage: () => "",
     nextPage: []
+  },
+  [config.UPDATE_CHECK_YOUR_ANSWERS_URL]: {
+    currentPage: config.UPDATE_CHECK_YOUR_ANSWERS_PAGE,
+    previousPage: () => config.OVERSEAS_ENTITY_REVIEW_URL,
+    nextPage: []
   }
```

Comment:
Think this is maybe a spill over from merge? This was added as part of UAR-137 on line 38